docs/roles/secretary.rst

```diff
@@ -9,22 +9,38 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for record maintenance and for keeping RITlug organized.
+Their primary tool as an eboard member is the Runbook.
```

Comment:
Small request: can you link to the Runbook repo here? It would be like this in reStructuredText:

```rest
is the `Runbook <https://github.com/RITlug/runbook>`__.
```